packages/valory/skills/registration_abci/behaviours.py

```diff
@@ -332,11 +289,33 @@ def request_tendermint_info(self) -> Generator[None, None, bool]:
             self.collection_complete = True
         return self.collection_complete
 
+    def format_genesis_data(
+        self,
+        collected_agent_info: Dict[str, Any],
+    ) -> Dict[str, Any]:
+        """Format collected agent info for genesis update"""
+
+        validators = []
+        for i, validator_config in enumerate(collected_agent_info.values()):
+            validator = dict(
+                address=validator_config["address"],
+                pub_key=validator_config["pub_key"],
+                power=self.params.voting_power,
+                name=f"node{i}",
+            )
+            validators.append(validator)
+
+        genesis_data = dict(
+            validators=validators,
+            genesis_config=self.params.genesis_config,
+        )
+        return genesis_data
+
     def request_update(self) -> Generator[None, None, bool]:
         """Make HTTP POST request to update agent's local Tendermint node"""
 
         url = self.tendermint_parameter_url
-        genesis_data = format_genesis_data(self.registered_addresses)
+        genesis_data = self.format_genesis_data(self.registered_addresses)
```

Comment:
Is tendermint com server up to date to work with this?

Reply:
https://github.com/valory-xyz/open-autonomy/blob/673761b24807845d04dda18e1e15dbaca8150a86/deployments/Dockerfiles/tendermint/app.py#L163-L193